draft-rokui-ccamp-actn-wdm-pluggable-modelling.md

```diff
@@ -195,7 +187,7 @@ Included in the model are representations of configuration, states, and telemetr
 
 Drawing from existing IETF models and potentially complementing that with input from other standard or industrial forum models (ITU-T, OpenConfig , ONF TAPI etc.) , this model offers enhanced uniform structuring and naming.
 
-This draft introduces the concept of "Coherent Pluggable Manifest", where it represents the capabilities of pluggable, maintained in a repository. This document also covers gap analysis of current IETF drafts and other SDOs on coherent Pluggable attributes and provides the complete lifecycle of a coherent pluggable from operators' approval through viability assessment to deployment.
+This draft introduces the concept of "Coherent Pluggable Repository", where it represents the capabilities of pluggable, maintained in a repository. This document also covers gap analysis of current IETF drafts and other SDOs on coherent Pluggable attributes and provides the complete lifecycle of a coherent pluggable from operators' approval through viability assessment to deployment.
 
 This document also covers an analysis of the gap between current IETF drafts and the corresponding works in other standards bodies and industry. The lifecycle of a coherent pluggable from operators' approval, viability assessment to deployment and monitoring is also covered.
 
```

Comment:
we should be clear about what we are discussing in this draft:
- not all pluggables are coherent
- not all coherent pluggables are DWDM
- not all DWDM interfaces are pluggable
-